lib/rstfilter/exec_setup.rb

```diff
@@ -81,14 +81,16 @@ def self.record begin_line, begin_col, end_line, end_col
     }
   }
 
-  class RubyVM::InstructionSequence
-    RST_FILENAME = ENV['RSTFILTER_FILENAME']
-    RST_MOD_SRC = File.read(ENV['RSTFILTER_MOD_SRC_PATH'])
-    @found = false
-    def self.translate iseq
-      if !@found && iseq.path == RST_FILENAME && iseq.label == "<main>"
-        @found = true
-        RubyVM::InstructionSequence.compile RST_MOD_SRC, RST_FILENAME
+  if defined?(RubyVM::InstructionSequence)
+    class RubyVM::InstructionSequence
+      RST_FILENAME = ENV['RSTFILTER_FILENAME']
+      RST_MOD_SRC = File.read(ENV['RSTFILTER_MOD_SRC_PATH'])
+      @found = false
+      def self.translate iseq
+        if !@found && iseq.path == RST_FILENAME && iseq.label == "<main>"
+          @found = true
+          RubyVM::InstructionSequence.compile RST_MOD_SRC, RST_FILENAME
+        end
```

Comment:
I couldn't really figure out what this code was used for? Seems to run ok without it?

Reply:
This is hidden feature.

rstfilter can specify executor: `-e ruby` and we can specify like `bundle exec rake test` here, and `--no-filename` execute the process without the filename.

For example: `rstfilter foo_spec.rb -e 'bundle exec spec' --no-filename` execute `bundle exec spec`.

In this case, `rstfilter` executes `bundle exec spec`, and only results for for `foo_spec.rb` are shown.

This code is for translating `foo_spec.rb`.

So the conclusion, this lines are for `-e` option.

Reply:
`--no-filename` doesn't seem to be an option? It's in the README, but it's never mentioned in the code?

Reply:
LSP seems to work.